src/testmod/java/ladysnake/satintestcore/item/DebugItem.java

```diff
@@ -27,11 +28,14 @@
 import net.minecraft.world.World;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class DebugItem extends Item {
     private int debugMode;
     private final List<DebugCallback> callbacks = new ArrayList<>();
+    private Map<DebugCallback, String> callbackNames = new HashMap<>();
```

Comment:
I believe it would be better to make a `DebugAction` record that stores a callback and a name, so that we can keep a single list of objects.